src/components/content-header-card/content-header-card.module.css

```diff
@@ -0,0 +1,174 @@
+.contentHeaderCard {
+	padding: 0;
+
+	& .cardTop {
+		padding: 16px;
+		display: flex;
+
+		& .icon {
+			margin-right: 12px;
+			display: none;
+
+			@media (--dev-dot-tablet-up) {
+				display: block;
+			}
+		}
+
+		& .content {
+			flex-grow: 1;
+
+			& .upper {
+				display: flex;
+				gap: 16px;
+				flex-direction: column-reverse;
+				@media (--dev-dot-tablet-up) {
+					flex-direction: row;
+				}
+
+				& .left {
+					flex-grow: 1;
+
+					& .titleWrap {
+						display: flex;
+						flex-wrap: wrap;
+						align-items: baseline;
+						@media (--dev-dot-tablet-up) {
+							flex-wrap: unset;
+						}
+
+						& .title {
+							color: var(--token-color-foreground-strong);
+						}
+
+						& .attribution {
+							color: var(--token-color-foreground-faint);
+							margin-left: 4px;
+						}
+					}
+
+					& .description {
+						margin-top: 2px;
+						color: var(--token-color-foreground-primary);
+					}
+				}
+
+				& .dropdown {
+					flex-shrink: 0;
+					width: 100%;
+					& > button {
+						width: 100%;
+					}
+
+					@media (--dev-dot-tablet-up) {
+						width: auto;
+						& > button {
+							width: auto;
+						}
+					}
+				}
+			}
+		}
+
+		& .lower {
+			display: flex;
+			flex-direction: column;
+			align-items: flex-start;
+			@media (--dev-dot-tablet-up) {
+				flex-direction: row;
+				align-items: flex-end;
+			}
+
+			& .badgeList {
+				margin-top: 16px;
+			}
+
+			& .note {
+				color: var(--token-color-foreground-faint);
+				flex-grow: 1;
+				text-align: right;
+				margin-top: 12px;
+				@media (--dev-dot-tablet-up) {
+					margin-top: 6px;
+				}
+			}
+		}
+	}
+
+	& .cardBottom {
+		padding: 12px 16px;
+		background: var(--token-color-surface-faint);
+
+		/* This is the color from --token-surface-base-box-shadow,
+		which the color value from it is not in a variable alone */
+		border-top: 1px solid rgba(101, 106, 118, 0.2);
+		border-radius: 0 0 6px 6px;
```

Comment:
🙋🏻 Are we able to use `overflow: hidden` on the parent element of this one, rather than set the border-radius with a hardcoded value? I think that would enable us to inherit the `border-radius` setting directly from `Card`.

Reply:
Ah, that is a good idea!

Reply:
Going to resolve this, as it won't work 😢

Reply:
Bummer! Was worth a try 😁